blocks/models/huggingface/base.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, id: str, config: HuggingfaceConfig):
         self.config = config
         self.model: Optional[Union[Callable, Trainer]] = None
 
-    def preload(self):
+    def load_remote(self):
```

Comment:
In huggingface I still have to seperate local and remote loading explicitly. It is currently unified.